server/src/main/java/org/elasticsearch/index/engine/ThreadPoolMergeExecutorService.java

```diff
@@ -87,8 +87,8 @@ public class ThreadPoolMergeExecutorService {
     private ThreadPoolMergeExecutorService(ThreadPool threadPool) {
         this.executorService = threadPool.executor(ThreadPool.Names.MERGE);
         this.maxConcurrentMerges = threadPool.info(ThreadPool.Names.MERGE).getMax();
-        this.concurrentMergesFloorLimitForThrottling = maxConcurrentMerges * 2;
-        this.concurrentMergesCeilLimitForThrottling = maxConcurrentMerges * 4;
+        this.concurrentMergesFloorLimitForThrottling = 2;
```

Comment:
Can we put a comment here that this means to only decrease throttle rate when we submit a task and no other tasks are running?